hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DataStreamer.java

```diff
@@ -1263,14 +1265,18 @@ private boolean processDatanodeOrExternalError() throws IOException {
       packetSendTime.clear();
     }
 
-    // If we had to recover the pipeline five times in a row for the
+    // If we had to recover the pipeline exceed times which
+    // defined in maxPipelineRecoveryRetries in a row for the
```

Comment:
nit:
Looks some grammatical error, can we change to,
``
If we had to recover the pipeline more than the value
 defined by maxPipelineRecoveryRetries in a row for the
``

Reply:
Thanks @ayushtkn  for review, i have fixed the grammatical error.